spec/services/apply_api/import_application_spec.rb

```diff
@@ -40,6 +40,16 @@ module ApplyApi
           end
         end
       end
+
+      context "when the provider type is an HEI" do
+        before do
+          application_data["attributes"]["course"]["training_provider_type"] = "university"
```

Comment:
https://www.apply-for-teacher-training.service.gov.uk/api-docs/reference#course-object something amiss?

Reply:
https://github.com/DFE-Digital/apply-for-teacher-training/pull/5366/files#diff-3ad9ab668ae825f7b5ce2640bdc332a97e78ba13c30de65810501e6c0c574aa7R390-R411

Perhaps docs not updated yet?